packages/sites/src/sites.ts

```diff
@@ -24,23 +23,23 @@ export function fetchDomains(
     ...requestOptions
   };
 
-  return request(url, options).then(result => {
-    return result;
-  });
+  return request(url, options);
 }
 
 /**
- * Fetch the domains associated with a Hub Site.
+ * Fetch the domain associated with a Hub Site. Since a site may have a
+ * custom domain, in addition to a default domain, we will return the
+ * custom domain over the default domain.
+ *
  * @param siteId - Identifier of the Hub Site
  * @param requestOptions - request options that may include authentication
  * @returns A Promise that will resolve with the domains associated with the site.
  */
-export function fetchDomain(
+export function fetchSiteDomain(
```

Comment:
another breaking name change. (i'm okay with it. just sayin)

Reply:
Not sure why I changed this... reverted...